pkg/operator/operator/k8s.go

```diff
@@ -294,10 +325,12 @@ func TensorFlowPredictorContainers(api *spec.API) ([]kcore.Container, []kcore.Vo
 	return containers, volumes
 }
 
-func ONNXPredictorContainers(api *spec.API) []kcore.Container {
+func ONNXPredictorContainers(api *spec.API) ([]kcore.Container, []kcore.Volume) {
 	resourceList := kcore.ResourceList{}
 	resourceLimitsList := kcore.ResourceList{}
 	containers := []kcore.Container{}
+	apiPodVolumeMounts := defaultVolumeMounts()
+	volumes := DefaultVolumes()
```

Comment:
nit: not exactly the purpose of this PR, but could we rename `DefaultVolumes` to `defaultVolumes`? It doesn't appear to be used outside this package and we can always open it up if we want.